pages/products/product.html

```diff
@@ -93,8 +94,166 @@ <h3>Sort by</h3>
             </div>
           </div>  
             <div class="product-cards-display">
-                <!-- <h2>Products displayed</h2> --> 
-                
+                <div class="normal-card">
+                    <div class="badge-card basic-card">
+                        <div class="card-image-button">
+                          <img class="card-image" src="https://i.pravatar.cc/150?img=11" alt="card-image" width="250">
+                          <button class="btn btn-icon btn-heart"><i class="fa-regular fa-heart"></i></button>
+                          <span class="badge text-badge">New Off</span>
+                        </div>
+                        <div class="card-text-container">
+                          <div class="card-description" >
+                            <h4>Matte black Sunglass</h4>
+                            <p>size wide </p>
+                            <p>Rs.399 <span class="text-strike">Rs.799</span> <span>50%</span></p> 
+                          </div>
+                        <div class="card-buttons">
+                          <button class="btn btn-primary">Add to Cart</button>
+                          <button class="btn btn-secondary">Order now</button>
+                        </div>
+                     </div>
+                    </div>
+                </div>
+                <div class="normal-card">
+                    <div class="badge-card basic-card">
+                        <div class="card-image-button">
+                          <img class="card-image" src="https://i.pravatar.cc/150?img=11" alt="card-image" width="250">
```

Comment:
Can come up with a better alt description. It is a given that it is an image. You can make it more meaningful by adding the product name to it. Eg. card-product-sunglasses.

Reply:
sure will add that